examples/advanced/variables.tf

```diff
@@ -37,3 +37,9 @@ variable "access_tags" {
   description = "Optional list of access management tags to add to the SCC WP instance"
   default     = []
 }
+
+variable "ibmcloud_resource_controller_api_endpoint" {
```

Comment:
No need to expose this in the example - just hard code it in the terraform code. You can use public endpoint for the example